sign_oca/views/sign_oca_request.xml

```diff
@@ -32,11 +39,11 @@
                 <sheet>
                     <div class="oe_button_box" name="button_box">
                         <button
-                            name="preview"
-                            type="object"
                             class="oe_stat_button"
                             icon="fa-pencil"
                             states="sent,signed"
+                            type="object"
```

Comment:
Why? Info is already on the notebook of the record?

We can add the button to sign there and we avoid to do too many click. From the button, we could go to the documents to sign. WDYT?

Reply:
Sorry, it is already changed (previously there was another approach perhaps more confusing). 
- The existing smart-button is kept (groups are added to avoid error for "_basic_" users without sign permissions), 
- A Sign button (`sign.oca.request` form) is added to be able to sign (for example a "basic" user without sign permissions).